Examples/Io/HepMC3/src/HepMC3Writer.cpp

```diff
@@ -26,6 +26,10 @@
 #include <HepMC3/WriterGZ.h>
 #endif
 
+#ifdef HEPMC3_ROOT_SUPPORT
```

Comment:
Shall we pre-fix this with ACTS_HEPMC3 ... to make sure it's an ACTS-local definition?